README.md

```diff
@@ -9,7 +9,7 @@ Skia Submodule Status: chrome/m97 ([upstream changes][skia-upstream], [our chang
 
 ## Goals
 
-This project provides _up to date_ safe bindings that bridge idiomatic Rust with Skia's C++ API on desktop and mobile platforms, including GPU rendering backends for [Vulkan](https://en.wikipedia.org/wiki/Vulkan_(API)), [Metal](https://en.wikipedia.org/wiki/Metal_(API)), [OpenGL](https://en.wikipedia.org/wiki/OpenGL), and [Direct3D](https://en.wikipedia.org/wiki/Direct3D).
+This project provides _up to date_ safe bindings that bridge idiomatic Rust with Skia's C++ API on desktop and mobile platforms, including GPU rendering backends for [Vulkan](<https://en.wikipedia.org/wiki/Vulkan_(API)>), [Metal](<https://en.wikipedia.org/wiki/Metal_(API)>), [OpenGL](https://en.wikipedia.org/wiki/OpenGL), and [Direct3D](https://en.wikipedia.org/wiki/Direct3D).
```

Comment:
Sorry about these changes my editor plugin automatically reformatted the whole file. I thought that since this is a pure addition it's OK but let me know if you want me to remove these from the diff.

Reply:
Surrounding links with `<>` is fine. Probably even _more_ correct to prevent parsing ambiguities. We can also keep the whitespace changes below.